JavaScript/image-tagger/index.html

```diff
@@ -185,7 +185,19 @@ <h4 class="whitespace-md">A simple, scalable API for machine intelligence</h4>
           LEARN MORE
         </a>
       </div>
-    </div>
+		</div>
+		
+		<div class="col-md-9" style="padding: 1em; margin-bottom: 1em; background-color: #eee;">
+			<h3>Image Credits</h3>
+			<ul>
+				<li> <a rel="nofollow" href="https://commons.wikimedia.org/wiki/File:Terrier_mixed-breed_dog.jpg">Terrier mixed-breed</a>" by <a rel="nofollow" href="https://www.flickr.com/photos/95609644@N00">Chris Barber</a> is licensed under <a rel="nofollow" href="https://creativecommons.org/licenses/by/2.0/deed.en">CC BY 2.0</a></li>
```

Comment:
All of these are missing opening quotes on the image name:
<img width="709" alt="Screen Shot 2019-11-11 at 2 40 14 PM" src="https://user-images.githubusercontent.com/25832730/68626910-49042980-0491-11ea-937f-74ed78bfd04f.png">

Reply:
Good eyeballs! Updated and redeployed. 👍